src/components/inputs/rich-text-input/editor.js

```diff
@@ -18,7 +18,7 @@ const Editor = props => {
   const intl = useIntl();
   const ref = React.useRef();
 
-  const prevIsFocused = usePrevious(props.isFocused);
+  const prevIsFocused = usePrevious(props.editor.value.selection.isFocused);
```

Comment:
I removed the `isFocused` state variable, now we only use the internal slate isFocused. This leads to less unnecessary re-renders.